### PR TITLE
Make 88x71 fluid on mobile

### PR DESCRIPTION
**Pull request by regiskuckaertz**
cc @jbreckmckye 

Before:
picture 2

After:
picture 1

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/commercial/dfp/dfp-api.js [hunk 1]
```diff
@@ -486,9 +486,22 @@ define([
...
                 });
             });
-        }
+        },
+        '88,70': isFluid250('ad-slot--top-banner-ad'),
+        '88,71': isFluid250('ad-slot--top-banner-ad')
+        '88,88': isFluid250('ad-slot--commercial-component'),
     };
 
+    function isFluid250(className) {
```
Comment (jbreckmckye) [inline static/src/javascripts/projects/common/modules/commercial/dfp/dfp-api.js]: I'm not quite sure I follow this - are these adverts actually fluid250s? If so, can we not just make `fluid250.js` add the class to its container?

**Comment by regiskuckaertz** [comment]
Nope 😭  There are other creatives that need it. This is fairly urgent but obviously the whole naming/logic should be rethought.

**Comment by jbreckmckye** [comment]
👍

**Comment by jbreckmckye** [comment]
👍 if it's actually needed - there's an outstanding question whether our existing Fabric templates could handle this requirement.

**Pull request merged**